Tests/BootCamp.Chapter.Tests/UnitTest1.cs

```diff
@@ -6,9 +6,17 @@ namespace BootCamp.Chapter.Tests
     public class UnitTest1
     {
         [Fact]
-        public void Test1()
+        public void UserSelectSizeAndDirection_ShouldReturnInt()
         {
-            Assert.True(true);
+            //Arrange
+            string userInput = "8";
+            int expected = 8;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'expected'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5002934)